src/v2/guide/comparison.md

```diff
@@ -84,145 +84,145 @@ If you are a fan of CSS-in-JS, many of the popular CSS-in-JS libraries support V
+Vue lebih sedikit teropini dibanding Angular, menawarkan dukungan resmi untuk beragam sistem bangunan, tanpa ada batasan bagimana struktur aplikasi kalian. Banyak pengembang menikmati kebebasan ini, tetapi sebagian lebih memilih hanya memiliki satu cara untuk membangun aplikasi.
 
-### Learning Curve
+### Lengkung pembelajaran (Tingkat kesulitan pembelajaran)
 
-To get started with Vue, all you need is familiarity with HTML and ES5 JavaScript (i.e. plain JavaScript). With these basic skills, you can start building non-trivial applications within less than a day of reading [the guide](./).
+Untuk mulai menggunakan Vue, semua yang kalian butuhkan hanyalah keakraban dengan HTML dan ES5 JavaScript (seperti JavaScript polosan). Dengan skill dasar tersebut, kalian bisa mulai membuat aplikasi sederhana dengan waktu kurang dari sehari hanya dengan membaca [panduan](./).
 
-Angular's learning curve is much steeper. The API surface of the framework is huge and as a user you will need to familiarize yourself with a lot more concepts before getting productive. The complexity of Angular is largely due to its design goal of targeting only large, complex applications - but that does make the framework a lot more difficult for less-experienced developers to pick up.
+Lengkung pembelajaran Angular lebih terjal. Permukaan API dari framework sangat besar dan sebagai pengguna kalian butuh membiasakan diri kalian dengan banyak konsep sebelum semakin produktif. Kompleksitas dari Angular besar karena tujuan desainnya hanyalah aplikasi besar, dan komplex- tetapi hal tersebut tidak membuat framework tidak susah untuk dipelajari untuk pengembang yang kurang berpengalaman.
 
 ## Ember
 
-Ember is a full-featured framework that is designed to be highly opinionated. It provides a lot of established conventions and once you are familiar enough with them, it can make you very productive. However, it also means the learning curve is high and flexibility suffers. It's a trade-off when you try to pick between an opinionated framework and a library with a loosely coupled set of tools that work together. The latter gives you more freedom but also requires you to make more architectural decisions.
+Ember adalah framework berfitur-lengkap yang di desain menjadi berpendirian teguh. Ember penyediakan banyak konvensi yang mapan dan setelah kalian terbiasa dengannya, dapat membuat kalian sangat produktif. Tetapi, hal ini berarti lengkung pembelajaran menjadi tinggi dan fleksibilitas terbebani. Hal ini sebuah pertukaran ketika kalian mencoba untuk mengambil diantara framework yang berpendirian dan beberapa peralatan library yang longgar yang dapat bekerja sama. Yang terakhir memberikan kalian kebebasan lebi tetapi juga membutuhkan kalian untuk membuat keputusan arsitektur.
```

Comment:
```suggestion
Ember adalah framework berfitur-lengkap yang di desain dengan opini yang kuat. Ember penyediakan banyak konvensi yang mapan dan setelah kalian terbiasa dengannya, dapat membuat kalian sangat produktif. Tetapi, hal ini berarti lengkung pembelajaran menjadi tinggi dan fleksibilitas terbebani. Hal ini sebuah pertukaran ketika kalian mencoba untuk mengambil diantara framework yang berpendirian dan beberapa peralatan library yang longgar yang dapat bekerja sama. Yang terakhir memberikan kalian kebebasan lebi tetapi juga membutuhkan kalian untuk membuat keputusan arsitektur.
```

Reply:
From my perspective, the word "Highly Opinionated" is having a wide and easy to use with any build system. Thus the word Opinionated trying to tell us is the Ember framework is easy to use without any explicit structure (it is depending on each developer's opinion).
```suggestion
Ember adalah framework berfitur-lengkap yang di dirancang untuk memiliki opini yang bebas. Ember penyediakan banyak konvensi yang mapan dan setelah kalian terbiasa dengannya, dapat membuat kalian sangat produktif. Tetapi, hal ini berarti lengkung pembelajaran menjadi tinggi dan fleksibilitas terbebani. Hal ini sebuah pertukaran ketika kalian mencoba untuk mengambil diantara framework yang berpendirian dan beberapa peralatan library yang longgar yang dapat bekerja sama. Yang terakhir memberikan kalian kebebasan lebi tetapi juga membutuhkan kalian untuk membuat keputusan arsitektur.
```

Reply:
The opinion here is not the user's opinion, but the framework designer's. So it means Ember can be easily used as long as we follow the rule set by the designer (so it require explicit structure from the designer).

https://stackoverflow.com/questions/802050/what-is-opinionated-software